ansible_collections/arista/avd/molecule/eos_config_deploy_cvp/intended/structured_configs/cvp/cv_server.yml

```diff
@@ -1,95 +0,0 @@
----
```

Comment:
@tgodaA I get this error when executing ```make refresh-facts```. I need to check whats going wrong.
```
TASK [arista.avd.eos_config_deploy_cvp : generate intented variables] **********
fatal: [cv_server]: FAILED! => {"changed": false, "module_stderr": "/bin/sh: /home/ans_cvp/.ansible/tmp/ansible-tmp-1643297162.8272243-9223-4627770579155/AnsiballZ_inventory_to_container.py: $(which: bad interpreter: No such file or directory\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 126}
```

Reply:
Try to git pull -rebase and then try once again to run ```molecule converge -s eos_config_deploy_cvp```. If that doesn't change anything alternatively we have other option.

Honestly, I won't suggest this always but, remove those two cvp files from git commit and push the changes.

Reply:
The changes that you made shouldn't have changed those two files.

Reply:
Could we do a zoom session?

Reply:
Yeah sure!

Reply:
https://arista.zoom.us/j/95301374884

Reply:
The issue is that this scenario has
```yaml
ansible_python_interpreter: $(which python)
```
so if your environment does not contain a pointer from `python` to `python3` this will fail. We should change the molecule scenario to
```yaml
ansible_python_interpreter: $(which python3)
```
If you can search for it and fix the file in your PR it would be great. Thanks

Reply:
Done. Hope it fits.

Reply:
Yeah, thank you @ClausHolbechArista

Reply:
Karl, please run "pre-commit run -a" with in avd directory to check the lint errors.

Reply:
Done... no errors:

```
➜  (AVD 🐳) ansible-avd pre-commit run -a
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/pycqa/pylint.
[INFO] Initializing environment for https://github.com/adrienverge/yamllint.git.
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pycqa/pylint.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/adrienverge/yamllint.git.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Passed
Check for added large files..............................................Passed
Check for merge conflicts................................................Passed
Check for Linting error on Python files..................................Passed
Check for Linting error on YAML files....................................Passed
```

Reply:
Great!

Reply:
The same command will be running again on Github. So I asked you to check locally for your info.